src/components/UploadPhoto.js

```diff
@@ -1,19 +1,15 @@
 import "../styles/UploadPhoto.css";
-// import { BsUpload } from "react-icons/bs";
 
-function UploadPhoto({ id, name, value, onChange }) {
+function UploadPhoto({ id, name, onChange }) {
   return (
     <div className="upload">
-      {/* <label for="file" className="upload__label">
-        <BsUpload className="upload__icon" />
-        Choose a Photo
-      </label> */}
       <input
         type="file"
+        accept="image/*"
+        multiple
```

Comment:
I thought you decided to disable multiple image upload for now :)
If so I'd advise to remove this option